include/cfe.h

```diff
@@ -7,8 +7,8 @@
 #include <float.h>
 #include <string.h>
 #include <assert.h>
-
-#include "giuh.h"
+#include "../include/conceptual_reservoir.h"

```

Comment:
I would not recommend using this type of relative include.  Include the header as `#include "conceptual_reservoir.h"` and make sure your build system puts a proper include dir flag `-I include` on the compile line.

Reply:
good catch @hellkite500. I would not recommend either, probably I overlooked it.